src/QMCDrivers/QMCDriverFactory.h

```diff
@@ -47,6 +48,7 @@ class QMCDriverFactory
     bool enable_profiling   = false;
     std::string traces_tag  = "none";
     QMCRunType new_run_type = QMCRunType::DUMMY;
+    std::optional<EstimatorManagerInput> estimator_manager_input;
```

Comment:
As I mentioned above, I feel better not copying global estimator input here.

Reply:
removed